routes/web/authUser.php

```diff
@@ -19,11 +19,19 @@
     'EventParticipationController@destroy'
 )->name('events.withdraw-part');
 
-Route::get('/quizzes/{quiz}', 'QuizController@instructions')
+Route::middleware('verify_quiz')->group(function() {
```

Comment:
instead of grouping just add middleware on all three routes, keeping it simple, this will also keep code flat (less indentation levels)